LeagueToolkit/IO/OBJ/OBJFile.cs

```diff
@@ -69,7 +92,7 @@ public void Write(string fileLocation)
 
         public void Write(Stream stream, bool leaveOpen = false)
         {
-            using (StreamWriter sw = new StreamWriter(stream, Encoding.UTF8, 1024, leaveOpen))
+            using (StreamWriter sw = new StreamWriter(stream, Encoding.Default, 1024, leaveOpen))
```

Comment:
Is this the encoding issue that you mentioned, where e.g. windows' 3d viewer will not accept the utf8 encoding? Do you have any source as to what encoding is expected in obj files, or did you just experimentally find this to work? Because generally, I'd prefer ascii or utf-8 encoding over the default (which should be utf-16).

Reply:
Changed to `ASCII`.